Source/WebKit/Platform/IPC/ArgumentCoders.cpp

```diff
@@ -105,7 +105,7 @@ static inline std::optional<String> decodeStringText(Decoder& decoder, unsigned
     auto data = decoder.template decodeSpan<CharacterType>(length);
     if (!data.data())
         return std::nullopt;
-    return std::make_optional<String>(data);
+    return std::make_optional<String>(String::fromSpan(data));
```

Comment:
Should not need the `make_optional` any more after this change.

Reply:
OK.